index.html

```diff
@@ -21,6 +21,9 @@
     <link rel="manifest" href="./manifest.json">
     <script src="./src/scripts/pwainit.js"></script>
 
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Why was it needed to fetch from CDN? Don't we already have it as a dependency?

Reply:
I have removed
Thanks